tf-modules/cumulus/README.md

```diff
@@ -24,6 +24,7 @@ This module's outputs are listed in [ouputs.tf](./outputs.tf). Notable values th
 - **report_executions_sns_topic_arn** - The ARN of the SNS topic used for reporting the status of Cumulus workflow executions. You will need this ARN if you want to publish to this topic directly.
 - **report_granules_sns_topic_arn** - The ARN of the SNS topic used for reporting the ingest status of granules for Cumulus workflows. You will need this ARN if you want to publish to this topic directly.
 - **report_pdrs_sns_topic_arn** - The ARN of the SNS topic used for reporting the ingest status of PDRs for Cumulus workflows. You will need this ARN if you want to publish to this topic directly.
+- **background_queue_url** - The url of the background queue.
```

Comment:
This seems .... terse.    Can we be a bit more descriptive here?

Reply:
Yeah it's terse. I'll remove for `outputs.tf`

Reply:
67a94876f3